oaipmhTemplates/ariadne/ariadneOwner.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+    <aocat:has_owner count="+">
```

Comment:
As subtemplates are parsed as XML before being merged with the parent template (so the merge is done in an XML-correct way), they have to declare all namespaces used in them (like `aocat` here).